include/deal.II/multigrid/mg_transfer.h

```diff
@@ -24,6 +24,10 @@
 #include <deal.II/lac/block_sparsity_pattern.h>
 #include <deal.II/lac/trilinos_sparse_matrix.h>
 #include <deal.II/lac/la_parallel_vector.h>
+#include <deal.II/lac/petsc_parallel_vector.h>
+#include <deal.II/lac/petsc_parallel_sparse_matrix.h>
+
+#include <deal.II/lac/sparsity_tools.h>
```

Comment:
I guess this is not needed in the header any more. Can you please delete it here? You might want to add it to `mg_transfer_prebuild.cc` if not there yet.

Reply:
The sparsity tools inclusion should be moved, agreed. The other two are required for defining MatrixSelector<dealii::PETScWrappers::MPI::Vector> at lines 153+ in the same file.